monegros-restaurant-dianamonegrodiaz/src/monegros/restaurant/entities/Order.java

```diff
@@ -84,7 +84,9 @@ public Double getTotalAmount() {
 	public void setTotalAmount(Double totalAmount) {
         this.totalAmount = totalAmount;
     }
-    
+    public void sendOrderToCashier(Order myOrder, Cashier cashier) {
```

Comment:
Very very bad idea to do something like this inside an entiry

Reply:
Why is using that method a bad idea?